src/Lykke.AlgoStore.Core/Enumerators/AlgoInstanceStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.AlgoStore.Core.Enumerators
+{
+    public enum AlgoInstanceStatus
```

Comment:
I think we should use Status Model from Models NuGet Package

Reply:
Ok, as we agreed, I will switch to using the Models NuGet Package from template project - entities, repo, etc.